File: projetandroid/app/src/main/java/com/example/testcompose/objects/CurrentForecast.kt
package com.example.testcompose.objects

import java.io.Serializable

data class CurrentForecast(
    val id: Int = 0,
    val description: String = "",
    val main: String = "",
    val temperature: Double = 0.0,
    val ville: String = "",
    val vitesse_vent: Double = 0.0
) : Serializable

File: projet_polypoke_server_nodejs/objects/CurrentForecast.js
class CurrentForecast {

    constructor(id, ville, temperature, main, description, vitesse_vent) {
        this.id = id; // pour gérer les icônes !
        this.ville = ville; // London
        this.temperature = temperature; // °C
        this.main = main; // Vent
        this.description = description; // Quelques rafales de vent
        this.vitesse_vent = vitesse_vent; // km/h

    }

    toJSON () {
        return {
            'id': this.id,
            'ville': this.ville,
            'temperature': this.temperature,
            'main': this.main,
            'description' : this.description,
            'vitesse_vent': this.vitesse_vent,
        };
      }


}

module.exports = CurrentForecast;